visibility/test/test_sun_position/TestSunPositionAndHillshade.py

```diff
@@ -23,15 +23,29 @@
 import traceback
 import TestUtilities
 import os
-import datetime
+import time
+import logging
+import unittest
 
+import UnitTestUtilities
 import SunPositionAndHillshadeUnitTest
 
 try:
-    # Call Unit Test code
+    #logFile=os.path.join(TestUtilities.toolboxesPath, 'SunLog.log')
+    logFile = "header.log"
```

Comment:
You may want to move this common logger setup stuff to a method in UnitTestUtilities.py also (& the method might just take the test name and do all of this)